common/test/acceptance/tests/studio/test_studio_help.py

```diff
@@ -47,3 +63,357 @@ def test_studio_help_links(self):
+
+            element_css = expected_link['element_css']
+            actual_link = get_element(page, element_css)
+            # Assert that link on DOM element are same as expected.
+            assert_links(self, expected_link, actual_link)
+            # Click the help link and assert that correct link is opened.
+            click_css(page, element_css, 0, False)
+            assert_help_is_open(self, expected_link['url'])
+
+
+class HomeHelpTest(StudioCourseTest):
+    """
+    Tests help links on 'Home'(Courses tab) page.
+    """
+    def setUp(self):  # pylint: disable=arguments-differ
+        super(HomeHelpTest, self).setUp()
+        self.home_page = HomePage(self.browser)
+
+    def test_course_home_help(self):
+        """
+        Scenario: Help link in navigation and side bar is working on 'Home'(Courses tab) page.
+        Given that I am on the 'Home'(Courses tab) page.
+        And I click the 'Help' in the navigation bar
+        Then help link should open.
+        And help url should contain 'getting_started/get_started.html'
+
+        Given that I am on the 'Home'(Courses tab) page.
+        And I want help about the courses
+        And I click the 'Getting Started with edX Studio' in the sidebar links
+        Then Help link should open.
+        And help url should contain 'getting_started/get_started.html'
+
+        """
+        expected_links = [
+            {
+                'element_css': NAV_HELP_CSS,
+                'text': 'Help',
+                'url': 'http://edx.readthedocs.org/projects/edx-partner-course-staff/'
+                       'en/latest/getting_started/get_started.html'
+            },
+            {
+                'element_css': '.bit li.action-item a',
+                'text': 'Getting Started with edX Studio',
+                'url': 'http://edx.readthedocs.org/projects/edx-partner-course-staff/'
+                       'en/latest/getting_started/get_started.html'
+            }
+        ]
+
+        for expected_link in expected_links:
+            self.home_page.visit()
+            element_css = expected_link['element_css']
+            actual_link = get_element(self.home_page, element_css)
+            # Assert that link on DOM element are same as expected.
+            assert_links(self, expected_link, actual_link)
+            # Click the help link and assert that correct link is opened.
+            click_css(self.home_page, element_css, 0, False)
+            assert_help_is_open(self, expected_link['url'])
+
+
+class NewCourseAndNewLibraryHelpTest(WebAppTest):
+    """
+    Test help links while creating new course and new library
+    """
+    def setUp(self):
+        super(NewCourseAndNewLibraryHelpTest, self).setUp()
+        self.auth_page = AutoAuthPage(self.browser, staff=True)
+        self.dashboard_page = DashboardPage(self.browser)
+        self.program_page = DashboardPageWithPrograms(self.browser)
+
+    def test_new_course_and_new_library_help(self):
```

Comment:
@Muddasser See my other comments regarding separate testcase classes, separate tests and moving css to page object file.